lib/addins/PulseLight.js

```diff
@@ -0,0 +1,47 @@
+ /* jshint esversion: 6, strict: true, node: true */
+ 
+'use strict';
+/**
+ * @type {HandlerPattern}
+ */
+var HandlerPattern = require('./handlerpattern.js');
+var log = require('debug')('PulseLight');
+
+/**
+ * @class A custom handler that always sends a 1 to KNX to iterate the same pulse trigger   
+ * @extends HandlerPattern
+ */
+class PulseLight extends HandlerPattern {
+
+	constructor(knxAPI) {
+		super(knxAPI); // call the super constructor first. Always.
+	}
+
+	/****
+	 * onKNXValueChange is invoked if a Bus value for one of the bound addresses is received
+	 * 
+	 */
+	onKNXValueChange(field, oldValue, knxValue) {
+        var newValue; //Value for Homekit
+        if(field === "On"){
+            if((knxValue? 1:0) !== (oldValue? 1:0)){
+               this.myAPI.setValue('On', newValue);
```

Comment:
you never set newvalue to anything, what will be sent to homekit?

Reply:
I think this code might be removed, as Homekit is the only source for setting the light there is no state to be send. Homekit remembers the state itself based on the signal send to KNX. There is no status to listen to for this PulseLight. I used existing code as base and tested it so it worked. Might be that there is code that could be removed / does not do anything special?